week1/DiceRoller/app/src/main/java/com/example/diceroller/MainActivity.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.diceroller
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.diceroller.ui.theme.DiceRollerTheme
+import java.lang.Math.random
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+          DiceRollerTheme {
+              DiceRollerApp()
+          }
+        }
+    }
+}
+
+@Composable
+fun DiceWithButtonAndImage(modifier: Modifier = Modifier
+    .fillMaxSize()
+    .wrapContentSize(Alignment.Center)){
+
+    var result =  remember { mutableStateOf(1) }
+
+    val imageResource = when (result.value) {
+        1 -> R.drawable.dice_1
+        2 -> R.drawable.dice_2
+        3 -> R.drawable.dice_3
+        4 -> R.drawable.dice_4
+        5 -> R.drawable.dice_5
+        else -> R.drawable.dice_6
+    }
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
```

Comment:
```suggestion
@Composable
fun DiceWithButtonAndImage(modifier: Modifier = Modifier) {

    var result =  remember { mutableStateOf(1) }

    val imageResource = when (result.value) {
        1 -> R.drawable.dice_1
        2 -> R.drawable.dice_2
        3 -> R.drawable.dice_3
        4 -> R.drawable.dice_4
        5 -> R.drawable.dice_5
        else -> R.drawable.dice_6
    }
    Column(
        modifier = modifier
            .fillMaxSize()
            .wrapContentSize(Alignment.Center),
        horizontalAlignment = Alignment.CenterHorizontally
    ) {
```

DiceWithButtonAndImage 함수의 인자로 modifier가 들어오더라도 항상 `fillMaxSize()`나 `wrapContentSize(Alignment.Center)`가 적용되도록 Column의 modifier에 적용시키는 건 어떨까요? (@yeoncheong 제가 생각한 게 맞을까요?)